### PR TITLE
Chores/ Add Give Me Some Credit

### PR DESCRIPTION
**Pull request by Philoso-Fish**
* Add dataset give me some credit and preprocessing
* Add folder for gitignore

### CODE REVIEW

**Review by indyfree — APPROVED** [review]
If we don't use the test set, i'd vote to remove it and rename the training set to a a normal name. Also at some point we should add documentation where we got the original dataset
